config/ota-contrib-declarations-production.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
Renaming these files will break deployment as they match the hostname of each instance.
I still think it's a good idea to rename these files but you will have to also rename hostnames on each instances.
I also suggest to rename hostname like the following:
- ota-contrib-declarations --> contrib
- ota-dating-declarations --> dating
- ota-france-declarations  --> france
- ota-france-elections-declarations --> france-elections

Reply:
That would be really good indeed! And part of this unification effort 👍

Reply:
DONE

I added a new variable `config_file_name` because relying on the hostname of the machine (that needs a by-hand configuration) is too magical for me